apps/src/lib/util/urlHelpers.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Attempt to construct an absolute Pegasus url (that is,
+ * starting with https://code.org or the appropriate
+ * equivalent for the current environment) from a given
+ * relative url.  If we're already on pegasus we'll
+ * just return the relative url.
+ * @param {string} relativeUrl - should start with a
+ *   leading slash.
+ */
+export function pegasus(relativeUrl) {
+  if (window.dashboard && window.dashboard.CODE_ORG_URL) {
```

Comment:
`window.dashboard.CODE_ORG_URL` is provided by [application.js.erb](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/assets/javascripts/application.js.erb#L40) so it should be present everywhere on dashboard, but not on pegasus.